components/common/navigation/Navigation.tsx

```diff
@@ -0,0 +1,99 @@
+import { type ReactElement, cloneElement, useState } from 'react';
+
+import { useRouter } from 'next/router';
+
+import Stack from '@mui/system/Stack';
+import MenuIcon from '@mui/icons-material/Menu';
+import CloseIcon from '@mui/icons-material/Close';
+import Container from '@mui/material/Container';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import useScrollTrigger from '@mui/material/useScrollTrigger';
+
+import { useThemeMode } from '../../../hooks/use-theme-mode';
+
+import Button from '../buttons/Button';
+import LogoBlackIcon from '../icon-logo/LogoBlackIcon';
+import NavigationLinks from './NavigationLinks';
+
+import { AppBarStyled, LogoButtonContainer, NavigationDrawer } from './styled-components';
+
+interface Props {
```

Comment:
Please, provide a bit better naming. It's too general ad can be confused

Reply:
'+